src/pluginscript_class_metadata.lua

```diff
@@ -266,6 +266,9 @@ function export(metadata)
 	return prop
 end
 
+-- Another alias for `export` since it's a reserved keyword in
+-- YueScript
+exp = export
```

Comment:
Added a small alias for `export` since it's a reserved keyword for YueScript. There might be a better name for this alias, but couldn't find one for now 😅

I could also add an macro for this in YueScript so I can just use the `property` method instead.

Reply:
`exp` sounds more like an exponential function of math libraries. Maybe `exported` or `exported_property` is a better name. Idk, `exp` just doesn't sound like a good name to me.